funnel/templates/project.html.jinja2

```diff
@@ -206,20 +206,20 @@
 {% endblock %}
 
 {% block footerinnerscripts %}
-  <script src="{{ built_asset('project.js') }}" type="text/javascript"></script>
-  <script type="text/javascript">
-    $(function() {
-      var projectConfig = {};
+  {%- if project.primary_venue.has_coordinates -%}
```

Comment:
`projectInit` is for initialising map. To improve page performance, load leaflet assets if the project has a venue.